lib/core/network/entryPoint.js

```diff
@@ -153,8 +186,9 @@ class EntryPoint {
     }
 
     try {
-      this.protocols[client.protocol].leaveChannel(channel, connectionId);
-    } catch (e) {
+      this.protocols.get(client.protocol).leaveChannel(channel, connectionId);
+    }
+    catch (e) {
```

Comment:
Shouldn't this be a runtime error instead?

Reply:
Maybe the service shouldn't crash if the protocol fails to do any action, should it keep trying though?

Reply:
Yes that's why I think.
(ping @scottinet )

Reply:
Should not be a crash, and it should keep trying. Don't forget that protocols can be user-made, and that kind of error might happen if, for instance, this action is triggered when cleaning real-time rooms _after_ the connection has been dropped, so we're asking a protocol to register that a non-existing connection has left a channel. It might trigger an exception on that one, and that doesn't mean it's worth to stop sending further events of that kind to it.